Resources/Prototypes/_SpaceCats/Entities/Objects/Misc/pens.yml

```
@@ -0,0 +1,370 @@
+    - Pickaxe
+  - type: Sprite
+    sprite: _SpaceCats/Objects/pens.rsi
+    state: fountain_black
+  - type: MeleeWeapon
+    wideAnimationRotation: -45
+    damage:
+      types:
+        Piercing: 15
+  - type: Tool
+    qualities:
+      - Screwing
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)